src/main/java/com/dawn/jat/illuminati/post/entity/PostEntity.java

```diff
@@ -16,7 +15,7 @@
 @Data
 @Document(collection = "post")
 public class PostEntity {
-    @Id
+    @Id @NonNull
```

Comment:
when creating new Entity I dont thing we have a attribute Id 🤔 So NonNull here is a little bit extreme I think

Reply:
Removed @NonNull